devolving_music/lib/Song_Scores.py

```diff
@@ -0,0 +1,32 @@
+from devolving_music.lib.elo_scoring import *
+class Song_Scores():
+    
+    def __init__(self,Song):
+        self._song_submision=Song
```

Comment:
```suggestion
        self._song_submission=Song
```

Typo. Change all usages as well.